src/stores/NotificationsController.ts

```diff
@@ -23,6 +23,12 @@ export const subscribeToNotifications = async (uuid: string = '') => {
           case NotificationsTypes.SUBSCRIBE_FAILED:
             window.location.href = addQueryParams(window.location.href, { u: 'payment-error', message: notification.resource?.message });
             break;
+          case NotificationsTypes.ACCOUNT_LOGOUT:
+            if (notification.resource?.reason === 'sessions_limit') {
+              window.location.href = addQueryParams(window.location.href, { u: 'simultaneous-logins' });
```

Comment:
I realize now... by using `window.location.href = 'new url'` the page actually refreshes. This doesn't result in a history push. Are you sure this is working as intended?